src/actions/index.js

```diff
@@ -0,0 +1,9 @@
+import { FETCH_CONTACTS } from './types';
+import data from '../data/data';
+
+export function fetchContacts() {
+  return {
+    type: FETCH_CONTACTS,
+    payload: data
```

Comment:
payload is used if you need to pass the parameters you get to the `fetchContacts()` action to the reducer. As there are no parameters to `fetchContacts()`, you can import that `data` directly into the reducer file and use it there. And leave the `fetchContacts()` action only with type.